src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -189,19 +241,101 @@ export class QuerySummary extends Component {
     }
   }
 
+  private parseNoResultsFoundMessage(noResultsFoundMessage: string) {
+    if (this.isQuerySummaryTagInMessage(noResultsFoundMessage)) {
+      let messageSections = noResultsFoundMessage.split(QUERY_TAG);
+      let messageElements = Array<Dom>();
```

Comment:
For uniformity reasons, we would normally use `Dom[]` instead of `Array<Dom>`. Both are pretty much equivalent though.

Reply:
No problem! Just changed it!

Reply:
👍